workflow/controller/controller_test.go

```diff
@@ -301,6 +301,31 @@ func withAnnotation(key, val string) with {
 	return func(pod *apiv1.Pod) { pod.Annotations[key] = val }
 }
 
+// createRunningPods creates the pods that are marked as running in a given test so that they can be accessed by the
+// pod assessor
+func createRunningPods(ctx context.Context, woc *wfOperationCtx, with ...with) {
```

Comment:
Useful utility testing function to create pods in the client that are supposed to be running according to the test workflow status.

Reply:
unused? delete?

Reply:
Only the `with` parameter was unused. Removed.